<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileWindowClass</class>
 <widget class="QWidget" name="FileWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>351</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>406</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelSpeed">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>0 B/sec</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTableView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>ToolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Add files</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/start.png</normaloff>:/icons/start.png</iconset>
   </property>
   <property name="text">
    <string>Start uploading</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/qt.png</normaloff>:/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionShow_links_to_files">
   <property name="icon">
    <iconset resource="upcoder.qrc">
     <normaloff>:/icons/internet.png</normaloff>:/icons/internet.png</iconset>
   </property>
   <property name="text">
    <string>Show links to files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="upcoder.qrc"/>
 </resources>
 <connections/>
</ui>
